src/kubernetes/client/mod.rs

```diff
@@ -57,10 +58,12 @@ impl Client {
             base,
             tls_options,
             token,
+            proxy,
         } = config;
 
         let tls_settings = TlsSettings::from_options(&Some(tls_options))?;
-        let inner = HttpClient::new(tls_settings)?;
+        let proxy = ProxyConfig::from_env().merge(&proxy);
+        let inner = HttpClient::new(tls_settings, &proxy)?;
```

Comment:
Is this right? How is the global config file proxy config injected here? Does `config.proxy` already take the global settings into account?

Reply:
That's a good point. I went too fast on these.